tests/serverpod_test_server/test_integration/database_operations/default/datetime_default_test.dart

```diff
@@ -0,0 +1,225 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
```

Comment:
Suggestion: See comments on the dart default tests and make the same changes here.